bark/runtime/scenario/scenario_generation/config_readers/observer_model_config_readers.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 fortiss GmbH
+#
+# Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+# Tobias Kessler
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+
+
+import numpy as np
+import math
+
+from bark.runtime.scenario.scenario_generation.config_readers.config_readers_interfaces import ConfigReaderObserverModel
+from bark.runtime.commons.parameters import ParameterServer
+from bark.core.models.observer import ObserverModelNone, ObserverModelParametric
+
+
+# observed worlds with no perturbations
+class ObserverModelNoneReader(ConfigReaderObserverModel):
+  # returns a observer model none for the world
+  def create_from_config(self, config_param_object,  **kwargs):
+    params = ParameterServer()
+    observer_model = ObserverModelNone(params)
+    return observer_model, {}, config_param_object
+
+# observed worlds with parametric perturbations
+class ObserverModelParametricReader(ConfigReaderObserverModel):
+  # returns a parametric observer model for the world
+  def create_from_config(self, config_param_object, **kwargs):
+    params = ParameterServer()
+    # if "EgoStateDeviationDist" in config_param_object:
```

Comment:
@juloberno is there a convenient way to generate a distribution here? what in your point of view would be the best implementation solution?

Reply:
You can just create the ParametricObserver and pass the config_param_object, distributions are all internally managed by the observer